index.html

```diff
@@ -21,8 +22,11 @@
 
 <body>
   <fe-app></fe-app>
+    
 
   <script type="module" src="./src/fe-app.js"></script>
+    <script type="module" src="./src/Components/fe-lang-toggle/fe-lang-toggle.js"></script>
```

Comment:
No need to add files here, bcz it's not used in html so do import where ur going to use that component